pytest/tests/loadtest/locust/common/inscription.py

```diff
@@ -0,0 +1,76 @@
+import logging
+from concurrent import futures
+import random
+import string
+import sys
+import pathlib
+import typing
+from locust import events, runners
+import common.base as base
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[4] / 'lib'))
+
+import key
+from common.base import Account, Deploy, NearNodeProxy, NearUser, FunctionCall, INIT_DONE
+
+
+class MintInscription(FunctionCall):
+
+    def __init__(self,
+                 contract_account_id: str,
+                 sender: Account,
+                 tick,
+                 amt):
+        # Attach exactly 1 yoctoNEAR according to NEP-141 to avoid calls from restricted access keys
+        super().__init__(sender, contract_account_id, "inscribe", balance=0)
+        self.sender = sender
+        self.tick = tick
+        self.amt = amt
+
+    def args(self) -> dict:
+        return {
+            "p": "nrc-20",
+            "op": "mint",
+            "tick": self.tick,
+            "amt": str(int(self.amt))
+        }
+
+    def sender_account(self) -> Account:
+        return self.sender
+
+@events.init.add_listener
+def on_locust_init(environment, **kwargs):
+    base.INIT_DONE.wait()
+    # `master_funding_account` is the same on all runners, allowing to share a
+    # single instance of inscription contract.
+    funding_account = environment.master_funding_account
+    environment.inscription_account_id = f"inscription.{funding_account.key.account_id}"
+
+    # Only create inscription contract on master.
+    if isinstance(environment.runner, runners.WorkerRunner):
+        return
+
+    node = base.NearNodeProxy(environment)
+    funding_account = base.NearUser.funding_account
+    funding_account.refresh_nonce(node.node)
+
+    account = base.Account(
+        key.Key.from_seed_testonly(environment.inscription_account_id))
+    node.prepare_account(account, funding_account, 50000,
+                         "create contract account")
+    node.send_tx_retry(
+        base.Deploy(
+            account,
+            environment.parsed_options.inscription_wasm,
+            "inscription",
+        ), "deploy inscription contract")
```

Comment:
Ah I think I need to perform a "deploy" action before minting the inscription.
Something like:
```
 '{"p": "nrc-20","op": "deploy", "tick": "inscription-name-here", "amt": "100000000000"}'
 ```
 I need to understand inscriptions better...

Reply:
you mean you were able to invoke the contract without deploying? maybe because it was already deployed? though, you are using a new custom account here to call the contract, hmm.

Reply:
AFAIU inscription is like a virtual token, I have to call a "create token" method before calling "mint token"
https://www.gate.io/learn/articles/how-to-get-started-with-inscriptions/1178

Reply:
Alright added a transaction which deploys the inscription